github/GithubApp.ts

```diff
@@ -162,7 +162,7 @@ export class GithubApp extends App {
             configuration.slashCommands.provideSlashCommand(ghCommand),
             this.getOauth2ClientInstance().setup(configuration),
         ]);
-        configuration.scheduler.registerProcessors([
+        await configuration.scheduler.registerProcessors([
```

Comment:
@Nabhag8848  Can you please put these `await` changes in a separate PR ?  It does not fit the purpose of this PR.

Reply:
Yep! thanks.